recipes/amqp-cpp/all/conanfile.py

```diff
@@ -74,10 +81,39 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
+        self._create_cmake_module_alias_targets(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+            {"amqpcpp": "amqpcpp::amqpcpp"}
+        )
+
+    @staticmethod
+    def _create_cmake_module_alias_targets(module_file, targets):
```

Comment:
Any chance this will be in Conan `tools`?

Reply:
yes https://github.com/conan-io/conan/pull/8533